src/ORM/Table.php

```diff
@@ -96,8 +96,10 @@
  * - `buildValidator(EventInterface $event, Validator $validator, string $name)`
```

Comment:
Why do we document these without the `Model.` prefix?

Reply:
These are the table methods. Your comment about conflating the methods and events is spot on. I will split up the two sections.